internal/filter/filter.go

```diff
@@ -11,6 +11,45 @@ import (
 // second argument.
 var ErrBadString = errors.New("filter.Match: string is empty")
 
+type patternPart struct {
+	pattern  string
+	isSimple bool
+}
+
+// Pattern represents a preparsed filter pattern
+type Pattern []patternPart
+
+func prepareStr(str string) ([]string, error) {
+	if str == "" {
+		return nil, ErrBadString
+	}
+
+	// convert file path separator to '/'
+	if filepath.Separator != '/' {
+		str = strings.Replace(str, string(filepath.Separator), "/", -1)
+	}
```

Comment:
str = filepath.ToSlash(str) is equivalent but can be compiled away entirely on Unix.

Reply:
Thanks for the hint. Now the code coverage is at 98% :-)